core/typesetter.lua

```diff
@@ -566,7 +570,8 @@ SILE.defaultTypesetter = std.object {
     local lines = {}
     local nodes = self.state.nodes
 
-    for i = 1,#bp do local point = bp[i]
+    for i = 1, #bp do
```

Comment:
We could even rename bp to breakpoints.

Reply:
Done.

How about `nl` → `nodes`? I think you meant "nodes from this line" or something like that, but it's pretty opaque.

Reply:
Maybe next time. :-) I’m not at a computer until Monday so let’s just get this one in so we can merge a few others too.